src/dotnet/Chat.Service/UserLinksBackendExt.cs

```diff
@@ -0,0 +1,29 @@
+namespace ActualChat.Chat;
+
+internal static class UserLinksBackendExt
+{
+    public static async Task UpdateUserLink(
+        ICommander commander,
```

Comment:
maybe make it an extension method?

Reply:
I don't want make it extension method to keep ICommander interface api cleaner.